diff --git a/urdfpy/urdf.py b/urdfpy/urdf.py
index 14b0216..b5245e7 100644
--- a/urdfpy/urdf.py
+++ b/urdfpy/urdf.py
@@ -895,7 +895,7 @@ def color(self):
     @color.setter
     def color(self, value):
         if value is not None:
-            value = np.asanyarray(value).astype(np.float)
+            value = np.asanyarray(value).astype(np.float64)
             value = np.clip(value, 0.0, 1.0)
             if value.shape != (4,):
                 raise ValueError('Color must be a (4,) float')
@@ -2344,7 +2344,7 @@ def get_child_pose(self, cfg=None):
             raise ValueError('Invalid configuration')
 
     def get_child_poses(self, cfg, n_cfgs):
-        """Computes the child pose relative to a parent pose for a given set of 
+        """Computes the child pose relative to a parent pose for a given set of
         configuration values.
 
         Parameters
